examples/tut-6-rpc/Makefile

```diff
@@ -7,8 +7,14 @@ clean:
 server/build/%.main.mjs: server/%.rsh
 	$(REACH) compile $^ main
 
+# TODO: ditch this once `@reach-sh/rpc-client` package is available on NPM
```

Comment:
My understanding is that we won't actually ditch this approach, because we always want CI to run this example against the in-repo version of the library